bench.go

```diff
@@ -1,14 +1,40 @@
 package main
 
-import "time"
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"os"
+	"time"
+
+	"github.com/prometheus/client_golang/api"
+	v1 "github.com/prometheus/client_golang/api/prometheus/v1"
```

Comment:
v1?

Reply:
The latest offical prometheus used this as an example...